c/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+cmake_minimum_required(VERSION 3.9)
+
+project(libblake3 VERSION 0.1.0 DESCRIPTION "BLAKE3 C implementation")
+enable_language(C ASM)
+
+include(GNUInstallDirs)
+
+option(BLAKE3_STATIC ON)
```

Comment:
It is best practice to use the CMake built-in [`BUILD_SHARED_LIBS`] option instead of custom options and/or target suffixes.

[`BUILD_SHARED_LIBS`]: https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html

Reply:
I reckon I tried it and it didn't work for some reason. I'm sorry but I can't recall why exactly, it was a while ago.